packages/web-components/src/components/cbp-tooltip/cbp-tooltip.scss

```diff
@@ -133,124 +133,136 @@ cbp-tooltip {
   --cbp-tooltip-caret-rotation: 45deg;
   --cbp-tooltip-caret-top: calc(50% - var(--cbp-tooltip-caret-offset));
   --cbp-tooltip-caret-left: calc(0% - var(--cbp-tooltip-caret-offset))
 }
 
-&[alignment='right-bottom'] div[role=tooltip]{
-  --cbp-tooltip-placement-top: calc(-150% + var(--cbp-tooltip-placement-offset, 0px));
+&[alignment='right-bottom']{
+  --cbp-tooltip-placement-top: unset;
   --cbp-tooltip-placement-left: calc(100% + var(--cbp-tooltip-placement-gap, 5px));
+  --cbp-tooltip-placement-bottom: 0%;
   --cbp-tooltip-placement-transform-x: 0;
   --cbp-tooltip-placement-transform-y: 0;
   --cbp-tooltip-caret-rotation: 45deg;
-  --cbp-tooltip-caret-top: calc(75% - var(--cbp-tooltip-caret-offset));
-  --cbp-tooltip-caret-left: calc(0% - var(--cbp-tooltip-caret-offset))
+  --cbp-tooltip-caret-top: calc(100% - var(--cbp-tooltip-caret-size) - var(--cbp-tooltip-caret-space));
+  --cbp-tooltip-caret-left: calc(0% - var(--cbp-tooltip-caret-offset));
 }
 
-&[alignment='bottom-left'] div[role=tooltip]{
+&[alignment='bottom-left']{
   --cbp-tooltip-placement-top: calc(100% + var(--cbp-tooltip-placement-gap, 5px));
   --cbp-tooltip-placement-left: calc(0px + var(--cbp-tooltip-placement-offset, 0px));
+  --cbp-tooltip-placement-bottom: unset;
   --cbp-tooltip-placement-transform-x: 0;
   --cbp-tooltip-placement-transform-y: 0;
   --cbp-tooltip-caret-rotation: 135deg;
-  --cbp-tooltip-caret-left: var(--cbp-tooltip-caret-offset);
+  --cbp-tooltip-caret-left: calc(0% + var(--cbp-tooltip-caret-space));
   --cbp-tooltip-caret-top: calc(0% - var(--cbp-tooltip-caret-offset))
 }
 
-&[alignment='bottom-center'] div[role=tooltip]{
+&[alignment='bottom-center']{
   --cbp-tooltip-placement-top: calc(100% + var(--cbp-tooltip-placement-gap, 5px));
   --cbp-tooltip-placement-left: 50%;
+  --cbp-tooltip-placement-bottom: unset;
   --cbp-tooltip-placement-transform-x: -50%;
   --cbp-tooltip-placement-transform-y: 0;
   --cbp-tooltip-caret-rotation: 135deg;
   --cbp-tooltip-caret-left: calc(50% - var(--cbp-tooltip-caret-offset));
   --cbp-tooltip-caret-top: calc(0% - var(--cbp-tooltip-caret-offset))
 }
 
-&[alignment='bottom-right'] div[role=tooltip]{
+&[alignment='bottom-right']{
   --cbp-tooltip-placement-top: calc(100% + var(--cbp-tooltip-placement-gap, 5px));
   --cbp-tooltip-placement-left: calc(100% + var(--cbp-tooltip-placement-offset, 0px));
+  --cbp-tooltip-placement-bottom: unset;
   --cbp-tooltip-placement-transform-x: -100%;
   --cbp-tooltip-placement-transform-y: 0;
   --cbp-tooltip-caret-rotation: 135deg;
-  --cbp-tooltip-caret-left: calc(100% - (var(--cbp-tooltip-caret-offset) * 3));
+  --cbp-tooltip-caret-left: calc(100% - (var(--cbp-tooltip-caret-size) + var(--cbp-tooltip-caret-space)));
   --cbp-tooltip-caret-top: calc(0% - var(--cbp-tooltip-caret-offset))
 }
 
-&[alignment='left-top'] div[role=tooltip]{
-  --cbp-tooltip-placement-top: calc(125% + var(--cbp-tooltip-placement-offset, 0px));
+&[alignment='left-top']{
```

Comment:
This case is still not working right and still has magic numbers. `--cbp-tooltip-placement-top` should be at 100% (plus modifiers) and  `--cbp-tooltip-placement-transform-y` at 0%

Reply:
updating, small note: looks better(aligns with top of control) for top @ 0% with the transform-y @ 0%